### ISSUE TITLE
where are the exchange rates put/written ?

### ISSUE THREAD

**Issue by shirishag75** [Qalculate/qalculate-gtk#49]
I was expecting the exchange rates to be put in either ~/.config/qalculate or better yet ~/.cache as can be seen at https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html . It does seem you do honor the freedesktop specs so would make sense to have that smallish database which would constantly change therein or have a directory if more than one datasets would be volatile

**Comment by hanna-kn** [comment]
Updated exchange rates are saved in ~/.local/share/qalculate (the same place as other local units, variables and functions). ~/.cache might indeed be a more appropriate place, but not ~/.config.

**Comment by shirishag75** [comment]
right, so please put it in ~/.cache whenever you make the changes, you are right about ~/.config not being the place as it would have dynamic content and ~/.config is usually more or less for static content.

**Comment by tastytea** [comment]
`$XDG_CACHE_HOME` is for non-essential data. I would argue that exchange rates are essential, because unit conversions will not work without it. The current path seems to be appropriate.

It is not uncommon to have `$XDG_CACHE_HOME` on a ram-disk or to wipe it at startup. Not being able to convert between units because my internet connection is down would be annoying.
